common/focuses/wc_physical_lifestyle_focuses.txt

```diff
@@ -2,131 +2,43 @@
 	lifestyle = physical_lifestyle
```

Comment:
Errors:
```[13:35:07][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: wc_can_start_new_lifestyle_tree_foreign_affairs_trigger, near line: 2
Unknown trigger type: wc_can_start_new_lifestyle_tree_majesty_trigger, near line: 3
Unknown trigger type: wc_can_start_new_lifestyle_tree_family_trigger, near line: 4
Unknown trigger type: wc_can_start_new_lifestyle_tree_strategy_trigger, near line: 5
Unknown trigger type: wc_can_start_new_lifestyle_tree_authority_trigger, near line: 6
Unknown trigger type: wc_can_start_new_lifestyle_tree_chivalry_trigger, near line: 7
Unknown trigger type: wc_can_start_new_lifestyle_tree_wealth_trigger, near line: 8
Unknown trigger type: wc_can_start_new_lifestyle_tree_domain_trigger, near line: 9
Unknown trigger type: wc_can_start_new_lifestyle_tree_duty_trigger, near line: 10
Unknown trigger type: wc_can_start_new_lifestyle_tree_skulduggery_trigger, near line: 11
Unknown trigger type: wc_can_start_new_lifestyle_tree_temptation_trigger, near line: 12
Unknown trigger type: wc_can_start_new_lifestyle_tree_intimidation_trigger, near line: 13
Unknown trigger type: wc_can_start_new_lifestyle_tree_medicine_trigger, near line: 14
Unknown trigger type: wc_can_start_new_lifestyle_tree_scholarship_trigger, near line: 15
Unknown trigger type: wc_can_start_new_lifestyle_tree_theology_trigger, near line: 16
" in file: " file: common/focuses/00_lifestyle_focuses.txt line: 33; can_start_new_lifestyle_tree_trigger" near line: 42
[13:35:07][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: wc_can_start_new_lifestyle_tree_foreign_affairs_trigger, near line: 34
" in file: "common/focuses/00_lifestyle_focuses.txt" near line: 35
[13:35:07][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: wc_lifestyle_tree_foreign_affairs_finished_trigger, near line: 39
" in file: "common/focuses/00_lifestyle_focuses.txt" near line: 39
[13:35:07][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: wc_can_start_new_lifestyle_tree_foreign_affairs_trigger, near line: 2
```

I think that's all of them, but they were repeating over and over in the logs.

Reply:
Errors related to missing triggers should be fixed when #633 is merged